### PR TITLE
Generate missing project files

### PR DESCRIPTION
**Pull request by jibrilg**

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -22,6 +22,6 @@ SPEC CHECKSUMS:
   OCMock: 721c11680ae2d65d67f40932777194daf7002781
   SquarePointOfSaleSDK: 1f69421c2d46e812cb65a0c6edd3e19aef339740
 
-PODFILE CHECKSUM: a679ea1c4a7a65af08221a21783dd323549bd283
+PODFILE CHECKSUM: e781ee37c527dfe47abbeca21610d209a98372ff
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```
[the remark this answers is not in the document]
Comment (tyten) [reply Podfile.lock:27]: are we not using 1.10.0 now?

[the remark this answers is not in the document]
Comment (jibrilg) [reply Podfile.lock:27]: yes i'm on 1.10.0....

[the remark this answers is not in the document]
Comment (jibrilg) [reply Podfile.lock:27]: Actually, cocoapods version 1.10.0 is specified in the Gemfile.lock. Branch was cut from conversion 5.51 [43e5424aa18] which has it as such

[the remark this answers is not in the document]
Comment (MikeSilvis) [reply Podfile.lock:27]: asdf

**Review by MikeSilvis — APPROVED** [review]

**Pull request merged**